src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -149,6 +149,18 @@ static IEnumerable<FunctionOverload> GetAlwaysPermittedOverloads()
                     .WithRequiredParameter("network", LanguageConstants.String, "String in CIDR notation containing an IP address range to be converted.")
                     .Build();
 
+                yield return new FunctionOverloadBuilder("buildUri")
+                    .WithReturnType(GetUriReturnType())
```

Comment:
no BuildUri returns a uri. This is the definition for the function in ARM repo  
public static Uri BuildUri(JObject value)

Reply:
> ah got it. I was confused with the way bicep works. I will change it

@anamikapan could you add a basic scenario or baseline test for the new functions? I think this would have most likely caught the issue with the type definition.

Reply:
Will do that